### ISSUE TITLE
Update Guava version - currently the latest released version 3.3.2 mentions Guava version that has known vulnerability.

### ISSUE THREAD

**Issue by michaelszymczak** [google/allocation-instrumenter#53]
The currently released artifact (3.3.2) mentions (`./META-INF/maven/com.google.guava/guava/pom.xml`) the version `31.1-jre` of guava that has a know vulnerability https://www.cve.org/CVERecord?id=CVE-2023-2976. 

It would be great to update it to the latest version of guava that fixed it (e.g. `32.0.1-jre`)

**Comment by cpovirk** [comment]
[Released](https://github.com/google/allocation-instrumenter/releases/tag/java-allocation-instrumenter-3.3.3) and [in Maven Central](https://repo1.maven.org/maven2/com/google/code/java-allocation-instrumenter/java-allocation-instrumenter/3.3.3/).

**Comment by michaelszymczak** [comment]
Indeed, the latest release 3.3.3 fetches the latest Guava, many thanks for the quick turnaround!
